CODE_OF_CONDUCT.md

```diff
@@ -31,6 +31,50 @@ Examples of unacceptable behavior by participants include:
 - Other conduct which could reasonably be considered inappropriate in a
   professional setting
 
+### Guide to Maintaining these Standards
+
+When interacting in our community, strive to maintain a friendly, empathetic tone. This includes, but is not limited to, interactions on Discord, GitHub, and Twitter.
+
+"Tone" refers to word, punctuation, and conversational choices that determine how your message _feels_ to others.
+
+Compare these two responses:
+
+_Question:_ Hey, I'm beginning with Astro. I tried to use `document` in my Astro page, but it says that the object doesn't exist. How can I manipulate the DOM?
+
+| Answer 1 | Answer 2 |
+| -- | -- |
+| You can't manipulate the DOM from an Astro file. Astro is a static site builder. | Hey, welcome to the community! <br/> Astro pages and components run on the server. Because of that, you won't have access to everything you're used to when writing JavaScript for the web. You're on the right track, though - you can access `document` from a `<script>` tag in your Astro file. <br/> Check out the docs here, and let me know if you need any more help! https://docs.astro.build/en/core-concepts/astro-components/#client-side-scripts |
+
+Answer 2 took more effort, but it made a significant difference. Answer 2
+
+- welcomed the new member into the community
+- used explanation points (conveying willingness to help) and friendly word choices
+- made them feel validated for asking the question ("You're on the right track")
+- provided an explanation that fit the person who asked the question. They specified that they were a beginner, so _Answer 2_ sought to explain things to someone who might not understand the implications of "static site builder".
+- made it clear that the community is willing to help, and asking is not a burden ("let me know if you need any more help!")
+
+Our community prides itself on being _positive_, _helpful_, and _inclusive_. If you have these goals in mind when interacting in the community, you're on the right track!
+
+- To stay positive:
+
+  - Make it clear that you appreciate the other person's message
```

Comment:
```suggestion
  - Make it clear that you appreciate the other person's message: even complaints can be valuable feedback
```
Just a thought! Take or leave it.

Also, I don't appreciate EVERY message... (insulting, crossing the line etc.) though it probably goes without saying, "We don't appreciate abuse, though!" Still, I see this and it stands out to me as "every message should be appreciated" and... that's not the case.

I'm trying to think if there's an appropriate substitute for "message" ("concerns"? No, sometimes they're just support questions or observations... "participation"? No, same as messages... "appreciate the other person getting involved?" ... "That you welcome the opportunity to ... help them? ... address their issue?" I don't have anything good at the moment, and am probably overthinking this. Just something that stood out.)